src/pyEQL/__init__.py

```diff
@@ -50,4 +50,9 @@
 # instantiated Store in memory, which should speed up instantiation of Solution objects.
 IonDB.connect()
 
+# Extend the UnitRegistry with custom units
+ureg.define("ppb = 1e-9 * mole / mole = parts_per_billion")
+ureg.define("ppm = 1e-6 * mole / mole = parts_per_million")
+ureg.define("% = 0.01 = percent")
+
```

Comment:
Good job figuring out how to hack the `UnitRegistry`! In principle this approach of using the unit registry could work, but in `get_amount` I've opted to manually translate the units into something `pint` understands, and that's the approach we should stick with in `preprocess_units`, so this block can be removed.